Documentation/workflow_example.Rmd

```diff
@@ -0,0 +1,223 @@
+
+```{r debias-ex, warning=FALSE}
+plan(sequential, workers=availableCores()-2)
+DebiasedATT(raw_data = sim_data, method_name = "SDID", num_placebos = 10) %>%
+  GapPlot(plot_title = "Debiased SDID ATT", plot_y_lab = "ATT")
+
+```
+
+
+## Auto Estimate
+For convenience, we provide a function that automatically determines the optimal method given the particular user inputted data (and the desired methods to try). Once again, this method relies heavily on the placebo treatment assignment generated based on the true data -- if the two are very similar, we can be more confident that the optimal method on the placebo data is likely the optimal method on the true data. In particular, taking a string of method names, the function generates a large number of placebo data and estimates (in parallel, for computational benefit) each method on each placebo dataset. Currently, the average mean squared error across the data sets (for a given horizon) is the criteria by which we determine the best method.^[We could also think about bias, though naively averaging the bias can be problematic as bias of -10 in time 1 and 10 in time 2 would look quite good. Perhaps take the average of the squared or absolute bias.] Future iterations will consider bias and perhaps even coverage, as this is often quite important in applied settings. Moreover, we plan to break the execution of the auto-estimator if certain diagnostic tests -- introduced in the first stages of the workflow -- are not adequately met (e.g. density of several features is quite different; inadequate coverage with respect to the PC plots; too many t-test are failed). ^[Thanks Jarod M for the suggestion.] 
```

Comment:
Here and elsewhere, consider using "---", not "--".  The former might produce an em-dash as opposed to an en-dash, which is used for ranges.